apps/OpenEnergyMonitor/myheatpump/myheatpump.js

```diff
@@ -610,12 +631,6 @@ function powergraph_load()
         } else {
             powergraph_series.push({label:"Flow rate", data:data["heatpump_flowrate"], yaxis:3, color:6});
         }
-        
-        
-    }
-    if (feeds["DHW_cylinderT"]!=undefined) {
-        data["DHW_cylinderT"] = feed.getdata(feeds["DHW_cylinderT"].id,view.start,view.end,view.interval,0,0,skipmissing,limitinterval);
-        powergraph_series.push({label:"DHW", data:data["DHW_cylinderT"], yaxis:2, color:5});
```

Comment:
Deleted this code because it was unused; there is no DHW_cylinderT feed defined.